Source/Charts/Renderers/DataRenderer.swift

```diff
@@ -61,12 +61,9 @@ internal struct AccessibleHeader {
                                 andData data: ChartData,
                                 withDefaultDescription defaultDescription: String = "Chart") -> NSUIAccessibilityElement
     {
-        let chartDescriptionText = chart.chartDescription.text ?? defaultDescription
-        let dataSetDescriptions = data.map { $0.label ?? "" }
-        let dataSetDescriptionText = dataSetDescriptions.joined(separator: ", ")
 
         let element = NSUIAccessibilityElement(accessibilityContainer: chart)
-        element.accessibilityLabel = chartDescriptionText + ". \(data.count) dataset\(data.count == 1 ? "" : "s"). \(dataSetDescriptionText)"
+        element.accessibilityLabel = "Precip data for next 7hours"
```

Comment:
Maybe we can pass in this value instead of having it hardcoded? also the 7hours will read ok? or does it need space?